lib/publiccloud/aws.pm

```diff
@@ -68,4 +69,11 @@ sub init {
     }
 }
 
+sub get_user_id {
+    zypper_call "in jq";
+    my $out = script_output("aws sts get-caller-identity | jq -r '.Account'");
```

Comment:
I put this here because is related directly with the AWS and not with the ECR, is something more generic that could be used in the future for other libraries that inherit from AWS one

Reply:
`
# echo "das" | jq
parse error: Invalid numeric literal at line 2, column 0
# echo $?
4
`

Reply:
I like the idea of checking the output, see https://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html it is a 12digit...

Reply:
And I would go one step further, that we add a member to `publiccloud::aws` called account_id and set it on `login()`.

Reply:
done, but it still pendent to remove the installation of jq